static/app/views/discover/table/tableView.spec.tsx

```diff
@@ -404,26 +409,30 @@ describe('TableView > CellActions', function () {
         tableData={{
           data: [
             {
+              id: '1',
               title: '/random/transaction/name',
               'p99(measurements.custom.kilobyte)': 444.3,
             },
           ],
+          // the types defined here are a mix of Record<string, ColumnType> & Record<string, Record<string, type>>
+          // which means that we cannot use the secondary Record<string> type to define the units as the ColumnType narrows
+          // down to a string literal type. We need to use the Record<string, Record<string, type>> type to define the units
+          // @ts-expect-error
           meta: {
             title: 'string',
             'p99(measurements.custom.kilobyte)': 'size',
             units: {
-              title: null,
               'p99(measurements.custom.kilobyte)': 'kilobyte',
             },
           },
         }}
         onChangeShowTags={onChangeShowTags}
       />
     );
-    await userEvent.hover(screen.getByText('444.3 KB'));
+    await userEvent.hover(screen.getByText(/444\.3 KB/));
```

Comment:
why this change?

Reply:
I'll revert it back, at some point the test broke and I was trying to see if the value was printed somewhere else